mipengine/resources/node_catalogue.json

```diff
@@ -0,0 +1,46 @@
+{
+  "globalNode": {
+    "nodeId": "global",
+    "rabbitmqURL": "127.0.0.1:5672",
+    "monetdbURL": "127.0.0.1:50000"
+  },
+  "localNodes": [
+    {
+      "nodeId": "local_node_1",
+      "rabbitmqURL": "127.0.0.1:5672",
+      "monetdbURL": "127.0.0.1:50000",
+      "data": {
```

Comment:
why key "data", use the word in the common project vocabulary "datasets". Appart from that "data" is too generic it can mean anything

Reply:
data includes pathologies and datasets (inside the pathologies)